### PR TITLE
Updated Chart Axis styling.

### PR DESCRIPTION
**Pull request by karatechops**
This PR brings Grommet's new charts styling inline with recent infographic projects. 

#### What does this PR do?
This PR adjusts font size and length of Axis component's tick marks. 

#### Where should the reviewer start?
This PR can be tested in the grommet-docs chart examples.

#### What testing has been done on this PR?
I've tested this styling update within the grommet-docs chart examples.

#### How should this be manually tested?
Grommet-docs chart examples is the best spot to test this PR as it provides multiple axis examples.

#### Any background context you want to provide?
This is the approved axis styling we have recently incorporated into infographics projects. 

#### What are the relevant issues?
Vertical axis labels longer than 6 characters will overflow the div. It is recommended to keep your Chart Axis labels to only a few characters.

#### Screenshots (if appropriate)
chart axis example 1
chart axis example 2

#### Do the grommet docs need to be updated?
Nope.

#### Should this PR be mentioned in the release notes?
This PR is a small styling update so I doubt that is a necessary action.

#### Is this change backwards compatible or is it a breaking change?
Nope.

### CODE REVIEW

File: src/scss/grommet-core/_objects.chart.scss [hunk 1]
```diff
@@ -1,5 +1,7 @@
 // (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
 
+$axis-length: 5px;
```
Comment (alansouzati) [inline src/scss/grommet-core/_objects.chart.scss]: can we use 6px? mayby halve(halve($inuit-base-spacing-unit)) ?

**Comment by ericsoderberghp** [comment]
Part of the chart Axis design is to allow callers to use richer labels, which is why the label properties are PropTypes.node. For instance, I could have a label that is:

**Comment by karatechops** [comment]
I was not aware the label supported the node proptype, that's interesting. Let me re-think this, I'll need to devise a mechanic to determine the size of the node and adjust the vertical Axis width based on that.

**Comment by ericsoderberghp** [comment]
Why do you need to explicitly manage the width of the Axis at all?

**Comment by karatechops** [comment]
The width constraint will be removed in the next commit. I was attempting to avoid [this bug](http://codepen.io/karatechops/pen/vXEPRA?editors=0010), when the label is longer than the axis' tick the tick line stretches to the full width. A potentially clean solution could be utilizing CSS ::before and ::after on grommetux-chart-axis__slot to place tick marks.

**Comment by ericsoderberghp** [comment]
I like the pseudo element approach. That seems semantically appropriate and safer.

**Pull request closed without merging**